corehq/apps/receiverwrapper/util.py

```diff
@@ -143,11 +143,13 @@ def get_commcare_version_from_appversion_text(appversion_text):
     '2.4.1'
     >>> get_commcare_version_from_appversion_text(u'संस्करण "2.27.8" (414593)')
     '2.27.8'
+    >>> get_commcare_version_from_appversion_text(u'CommCare Android, आवृत्ती" 2.44.5"(452680). ॲप वि.29635 कॉमर्स आवृत्ती2.44. बिल्ड452680, रोजी तयार केले:2019-01-17')
+    '2.44.3'
     """
     patterns = [
         r'version "([\d.]+)"',
         r'"([\d.]+)"\s+\(\d+\)',
-        r'"([\d.]+)"',
+        r'([\d.]+)"\(',
```

Comment:
what about `"\s*([\d.]+)\s*"` which allows whitespace before and after the version

Reply:
Sure